src/server/game/Scripting/ScriptMgr.cpp

```diff
@@ -32,7 +32,21 @@ namespace
     {
         for (auto const& [scriptID, script] : ScriptRegistry<T>::ScriptPointerList)
         {
-            delete script;
+            try
+            {
+                if (script)
+                {
+                    delete script;
```

Comment:
Concerning the crash, I suspect a situation where we can have here a dangling pointer. We can maybe force a nullptr reference for script variable, but I'm doubt if it wasn't handle otherwise in core.

Reply:
I find any to not ensure the dereferencing of script pointers by forcing a nullptr, and after few tests and a debugging  it seems to change nothing.